mixins/StoreMixin.js

```diff
@@ -76,9 +76,16 @@ var StoreMixin = {
   store_addListeners: function () {
     Object.keys(this.store_listeners).forEach(function (storeID) {
```

Comment:
i just realized that we call `Object.keys` on `this.store_listeners` which is an array. that's.... weird